src/pfs/pfs.proto

```diff
@@ -49,7 +49,7 @@ message RepoInfo {
 message RepoAuthInfo {
   // The callers access level to the relevant repo (e.g. may be OWNER even if
   // the user isn't an OWNER of the repo, if they're an admin for the cluster)
-  auth.Scope access_level = 1;
+  repeated auth.Permission permissions = 1;
```

Comment:
This may be another API that it may make sense to makeover in 2.0 (not necessarily in this PR, just an opportunity that occured to me). I think of the main consumer of this field as being the dash, which wants to gray out repos that the user can't access. A list of permissions would definitely work, but I wonder if there's a more UI-friendly thing it would make sense to return?

Reply:
I'll talk to the Dash 2.0 team about this - I do think they'll want to know the permissions so they can display appropriate UI elements?